2022-03-02 15:20:25,547 INFO    MainThread:11008 [wandb_setup.py:_flush():71] setting env: {}
2022-03-02 15:20:25,547 INFO    MainThread:11008 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-02 15:20:25,547 INFO    MainThread:11008 [wandb_init.py:_log_setup():371] Logging user logs to /home/sohui/code/DTC/code/wandb/run-20220302_152025-2lauy913/logs/debug.log
2022-03-02 15:20:25,548 INFO    MainThread:11008 [wandb_init.py:_log_setup():372] Logging internal logs to /home/sohui/code/DTC/code/wandb/run-20220302_152025-2lauy913/logs/debug-internal.log
2022-03-02 15:20:25,548 INFO    MainThread:11008 [wandb_init.py:init():404] calling init triggers
2022-03-02 15:20:25,548 INFO    MainThread:11008 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'mdoel': 'DTC', 'server': 57, 'image size': '112,112,80'}
2022-03-02 15:20:25,549 INFO    MainThread:11008 [wandb_init.py:init():449] starting backend
2022-03-02 15:20:25,549 INFO    MainThread:11008 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-02 15:20:25,572 INFO    MainThread:11008 [backend.py:ensure_launched():199] starting backend process...
2022-03-02 15:20:25,582 INFO    MainThread:11008 [backend.py:ensure_launched():204] started backend process with pid: 11215
2022-03-02 15:20:25,585 INFO    MainThread:11008 [wandb_init.py:init():458] backend started and connected
2022-03-02 15:20:25,589 INFO    MainThread:11008 [wandb_init.py:init():519] updated telemetry
2022-03-02 15:20:25,591 INFO    MainThread:11008 [wandb_init.py:init():549] communicating current version
2022-03-02 15:20:26,821 INFO    MainThread:11008 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-02 15:20:26,822 INFO    MainThread:11008 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-03-02 15:20:27,082 INFO    MainThread:11008 [wandb_init.py:init():592] starting run threads in backend
2022-03-02 15:20:27,440 INFO    MainThread:11008 [wandb_run.py:_console_start():1816] atexit reg
2022-03-02 15:20:27,441 INFO    MainThread:11008 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2022-03-02 15:20:27,441 INFO    MainThread:11008 [wandb_run.py:_redirect():1695] Redirecting console.
2022-03-02 15:20:27,446 INFO    MainThread:11008 [wandb_run.py:_redirect():1751] Redirects installed.
2022-03-02 15:20:27,446 INFO    MainThread:11008 [wandb_init.py:init():619] run started, returning control to user process
2022-03-02 15:20:54,689 INFO    MainThread:11008 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-03-02 15:20:54,690 INFO    MainThread:11008 [wandb_run.py:_restore():1758] restore
2022-03-02 15:20:57,575 INFO    MainThread:11008 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 588
  total_bytes: 588
}

2022-03-02 15:20:57,824 INFO    MainThread:11008 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 588
  total_bytes: 588
}

2022-03-02 15:20:58,078 INFO    MainThread:11008 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 588
  total_bytes: 588
}

2022-03-02 15:20:58,181 INFO    MainThread:11008 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 588
  total_bytes: 5155
}

2022-03-02 15:20:58,285 INFO    MainThread:11008 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 588
  total_bytes: 5155
}

2022-03-02 15:20:58,412 INFO    MainThread:11008 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 588
  total_bytes: 5155
}

2022-03-02 15:20:58,515 INFO    MainThread:11008 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5155
  total_bytes: 5155
}

2022-03-02 15:20:58,618 INFO    MainThread:11008 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5155
  total_bytes: 5155
}

2022-03-02 15:20:58,721 INFO    MainThread:11008 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5155
  total_bytes: 5155
}

2022-03-02 15:20:58,825 INFO    MainThread:11008 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5155
  total_bytes: 5155
}

2022-03-02 15:20:58,927 INFO    MainThread:11008 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5155
  total_bytes: 5155
}

2022-03-02 15:20:59,030 INFO    MainThread:11008 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5155
  total_bytes: 5155
}

2022-03-02 15:20:59,133 INFO    MainThread:11008 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5155
  total_bytes: 5155
}

2022-03-02 15:20:59,799 INFO    MainThread:11008 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5155
  total_bytes: 5155
}

2022-03-02 15:21:00,314 INFO    MainThread:11008 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5155
  total_bytes: 5155
}
local_info {
}

2022-03-02 15:21:01,525 INFO    MainThread:11008 [wandb_run.py:_append_files():2184] logging synced files
